src/main/java/com/twilio/converter/Converter.java

```diff
@@ -18,7 +18,7 @@ public class Converter {
      * @param map map to convert
      * @return converted JSON string
      */
-    public static String mapToJson(final Map<String, Object> map) {
+    public static String mapToJson(final Map<String, ? extends Object> map) {
```

Comment:
This line of code is added to support all object serialization which has a value of type Object or any object.